core/boards/nrf52840_pca10056-sub-slots-debug.dts

```diff
@@ -29,12 +29,13 @@
 		};
 
 		/*
-		 * Using last 4 pages of storage partition
+		 * Using 4 pages of scratch partition for OpenThread storage.
```

Comment:
I suggest you specify which pages the scratch is using.

Reply:
Done